src/containers/file-upload/FileUploadProvider.js

```diff
@@ -22,7 +22,9 @@ export default class FileUploadProvider extends PureComponent {
   render() {
     return (
       <Provider store={getStore()}>
-        <ConnectedFileUpload>{this.props.children}</ConnectedFileUpload>
+        <ConnectedFileUpload {...this.props}>
```

Comment:
Why this change?

Reply:
For simple passing the props to the actual component.
We can pass explicit props instead, then `className` is the one we would need in this PR.